PGM/src/main/java/tc/oc/pgm/mutation/types/uhc/BarebonesScenario.java

```diff
@@ -0,0 +1,96 @@
+package tc.oc.pgm.mutation.types.uhc;
+
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.inventory.CraftItemEvent;
+import org.bukkit.event.block.BlockBreakEvent;
+import org.bukkit.inventory.CraftingInventory;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.event.entity.PlayerDeathEvent;
+import tc.oc.pgm.match.Match;
+import tc.oc.pgm.mutation.Mutation;
+import tc.oc.pgm.mutation.types.UHCMutation;
+
+public class EnchantlessScenario extends UHCMutation.Impl {
+    public EnchantlessScenario(Match match, Mutation mutation) {
+        super(match, mutation);
+    }
+
+    // Disable drops for ores that are tiered above iron
+    @EventHandler(priority = EventPriority.HIGH)
+    public void onBlockBreak(BlockBreakEvent event) {
+        Block block = event.getBlock();
+
+        if (block.getType() == Material.REDSTONE_ORE) {
+            event.getItemDrop().remove();
+            block.getState().update();
+            event.getBlock().setType(Material.AIR);
```

Comment:
You don't need to set the block to air, or call a block update. If the event is not canceled, the block will just break naturally with no blocks.